plugins/poll/assets/javascripts/discourse/components/poll-results-standard.gjs

```diff
@@ -40,7 +40,7 @@ export default class PollResultsStandardComponent extends Component {
       const chosen = (this.args.vote || []).includes(option.id);
       option.percentage = per;
       option.chosen = chosen;
-      let voters = this.args.voters[option.id] || [];
+      let voters = this.args.isPublic ? this.args.voters[option.id] || [] : [];
```

Comment:
I know this isn't on you, but would it make sense to not return the voters from the serializer if the poll is private?

Reply:
Hey @nattsw I don't believe it is currently.   `preloaded_voters` is only included if the poll is public - the API I believe is ok in this regard.

Reply:
Actually I think I was confused by `voters` being returned as an `int` and it being a `[]` here... 😅